gui/app.py

```diff
@@ -24,18 +24,20 @@ def __init__(self, my_bot):
         self.configure_window()
         self.frame_storage = {}
 
+        self.bot = my_bot
```

Comment:
we will now access the bot through here instead of passing it down over and over again.